linux_os/guide/system/accounts/accounts-restrictions/account_expiration/ensure_shadow_group_empty/oval/shared.xml

```diff
@@ -0,0 +1,48 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure shadow group is empty") }}}
+    <criteria>
+      <criterion test_ref="tst_shadow_group_empty" comment="shadow group is empty" />
+      <criterion test_ref="tst_no_user_assigned_shadow_group" comment="no user is assigned to the shadow group" />
```

Comment:
Besides the line length, could the comment be more specific by informing the test is actually checking the primary group of users. Maybe `no user has the shadow as primary group`.

Reply:
thanks for the suggestion, added it now